### PR TITLE
Add shysank to capz maintainers

### PR DESCRIPTION
**Pull request by shysank**

### CODE REVIEW

**Comment by shysank** [comment]
cc @CecileRobertMichon

File: config/jobs/kubernetes-sigs/cluster-api-provider-azure/OWNERS [hunk 1]
```diff
@@ -10,11 +10,11 @@ approvers:
 - karataliu
 - khenidak
 - nader-ziada
+- shysank
 reviewers:
 - cpanato
 - juan-lee
 - mboersma
-- shysank
```
Comment (CecileRobertMichon) [inline config/jobs/kubernetes-sigs/cluster-api-provider-azure/OWNERS:17]: can you also please add @jackfrancis and @jsturtevant so this is up to date with our owners file?

Comment (shysank) [reply config/jobs/kubernetes-sigs/cluster-api-provider-azure/OWNERS:17]: done

**Review by CecileRobertMichon — COMMENTED** [review]
/lgtm
/approve

**Comment by jackfrancis** [comment]
lgtm, reminder to squash commits 🙇

**Pull request merged**